.github/workflows/schedule-full-ci-test-main.yml

```diff
@@ -0,0 +1,153 @@
+name: QMI Python CI full test runner
+
+on:
+  schedule:
+    # Schedule for every day at 7am
+    - cron: "0 7 * * *"
+
+env:
+  # minimum pylint code quality score (max. 10)
+  PYLINT_MIN_SCORE: "9.00"
+  # minimum code coverage (goal: 80%; max. 100%)
+  COVERAGE_MIN_PERC: "68"
+  # maximum code complexity (goal: <= 15; unbounded)
+  COMPLEXITY_MAX_SCORE: "38"
+  # minimum maintainability index (goal: >=20; max. 100)
+  MAINTAINABILITY_MIN_INDEX: "10.62"
+  SOURCE_DIRS: "qmi/"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python-latest
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          sudo apt-get update -qy
+          sudo apt-get install -y bc
+          python3 --version
+          pip install --upgrade pip
+          pip install -e '.[dev]'
+          pip install anybadge
+      - name: pylint
+        run: |
+          pip install pylint
+          chmod a+x scripts/run_analysis_pylint.sh
+          SCORE=$( scripts/run_analysis_pylint.sh )
+          echo $SCORE $PYLINT_MIN_SCORE
+          echo $SCORE > pylint_score
+          # bc evaluates to `1` if relation is true
+          exit $( echo "$SCORE < $PYLINT_MIN_SCORE" | bc )
+          SCORE=$( cat pylint_score )
+          anybadge -l pylint -v $SCORE -f pylint.svg 2=red 4=orange 8=yellow 10=green
+        continue-on-error: true
+      - name: upload pylint test results
+        uses: actions/upload-artifact@v3
+        with:
+          name: pylint-results
+          path: pylint.log
+        if: ${{ always() }}
+      - name: upload pylint test badge
+        uses: actions/upload-artifact@v3
+        with:
+          name: pylint-badge
+          path: pylint.svg
+        if: ${{ always() }}
+      - name: mypy
+        run: |
+          pip install mypy
+          mypy --namespace-packages $SOURCE_DIRS | tee mypy.log
+          if [ -n "$( tail -n 1 mypy.log | grep -e '^Succes' )" ]; then RESULT="passed"; else RESULT="failed"; fi
```

Comment:
`Succes` to `Success`

Reply:
This matching works adequately like this so no need to add the extra 's'.